spring-cloud-alibaba-sentinel/src/main/java/com/alibaba/cloud/sentinel/custom/SentinelProtectInterceptor.java

```diff
@@ -65,6 +65,24 @@ public ClientHttpResponse intercept(HttpRequest request, byte[] body,
 		if (hostResource.equals(hostWithPathResource)) {
 			entryWithPath = false;
 		}
+		Method urlCleanerMethod = BlockClassRegistry.lookupUrlCleaner(
+				sentinelRestTemplate.urlCleanerClass(),
+				sentinelRestTemplate.urlCleaner());
+		if (urlCleanerMethod != null) {
+			try {
+				hostWithPathResource = (String) urlCleanerMethod.invoke(null,
```

Comment:
no need to use native method invoke, use `methodInvoke`

Reply:
But `methodInvoke` returns `ClientHttpResponse` while `urlCleanerMethod` returns `String`.  Do I need to create another function like `methodInvoke` which returns `String`?

Reply:
It return `Object`, we can use do some type cast.

Reply:
So I just need to modify `methodInvoke` to make it return Object and do type casting with `ClientHttpResponse` and `String`?

Reply:
yes 👍